clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -804,6 +890,42 @@ class SyclKernelFieldChecker
   bool IsInvalid = false;
   DiagnosticsEngine &Diag;
 
+  // Check whether the object should be disallowed from being copied to kernel.
+  // Return true if not copyable, false if copyable.
+  bool checkNotCopyableToKernel(const FieldDecl *FD, const QualType &FieldTy) {
+    if (FieldTy->isArrayType()) {
+      if (const auto *CAT = dyn_cast<ConstantArrayType>(FieldTy)) {
+        QualType ET = CAT->getElementType();
+        return checkNotCopyableToKernel(FD, ET);
+      } else
```

Comment:
```suggestion
      }
```
Please, [don't use else after return](https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return).